plugins/@grouparoo/sqlite/__tests__/data/records.csv

```diff
@@ -3,8 +3,8 @@ id,first_name,last_name,email,gender,ip_address,ios_app,android_app,vip,ltv,date
 2,Cacilie,Eate,ceate1@example.com,Female,253.146.41.201,false,true,false,94.36,2020/02/02,2020/02/02 12:13:14
 3,Gretel,Groucock,ggroucock2@example.com,Female,211.89.110.134,true,false,true,668.30,2020/02/03,2020/02/03 12:13:14
 4,Amata,Cotesford,acotesford3@example.com,Female,193.9.240.246,true,true,true,489.00,2020/02/04,2020/02/04 12:13:14
-5,Lira,Johnston,ljohnston4@example.com,Female,53.161.58.99,false,true,false,823.16,2020/02/05,2020/02/05 12:13:14
-6,Deni,Scalia,dscalia5@example.com,Female,138.131.148.74,true,false,true,273.98,2020/02/06,2020/02/06 12:13:14
+5,Lira,"""Airquotes"" Johnston",ljohnston4@example.com,Female,53.161.58.99,false,true,false,823.16,2020/02/05,2020/02/05 12:13:14
+6,Deni,O‘Hara,dscalia5@example.com,Female,138.131.148.74,true,false,true,273.98,2020/02/06,2020/02/06 12:13:14
```

Comment:
Nice.

Reply:
Although... I don't see those values tested for in a test?  I would have expected `expect(users[6].lastName).toBe()` somewhere

Reply:
true, will fix